code_generation/producers/scalefactors.py

```diff
@@ -1,36 +1,66 @@
 import code_generation.quantities.output as q
 from code_generation.producer import Producer, ProducerGroup
+from code_generation.producer import ExtendedVectorProducer
+
 
 ############################
 # Muon ID, ISO SF
-# The readout is done via RooWorkspaces
+# The readout is done via RooWorkspaces or correctionlib (with *_UL)
 ############################
 
+Muon_1_ID_SF_old = Producer(
```

Comment:
rename `old` to `RooWorkspace` so something like `calefactor::muon::id_rooworkspace`

Reply:
and keep `correctionlib` version of the function without a additional name to make clear, that this is the default

Reply:
Done